src/screens/BackupMnemonic/VerificationSuccessScreen.tsx

```diff
@@ -46,7 +46,7 @@ const VerificationSuccessScreen = ({ navigation, ...props }: VerificationSuccess
       />
     </ScreenSection>
     <ScreenSection>
-      <Button variant="highlight" title="Return to my wallet" onPress={() => resetNavigationState()} />
+      <Button variant="highlight" title="Return to my wallet" onPress={() => resetNavigation(navigation.getParent())} />
```

Comment:
We need to get the parent because we are in a sub-navigation. TypeScript will properly complain if the wrong navigator is passed, which makes our code safer.